### PR TITLE
Adds email notification if failure occurs

### PR DESCRIPTION
**Pull request by ives1227**
**Uses the lts-mqemailnotifier to send a message to the queue to trigger the dais-notifier**
* * *

**GitHub Issue**: https://jira.huit.harvard.edu/browse/LTSEPADD-93

* Other Relevant Links (Mailing list discussion, related pull requests, etc.)
Other Relevant Links (Mailing list discussion, related pull requests, etc.)
MQ Email Notifier Module: https://github.com/harvard-lts/mq_email_notifier

# How should this be tested?

Follow the testing instructions here: https://github.com/harvard-lts/drs-translation-service#testing

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? yes
- integration tests? no

### CODE REVIEW

**Comment by awoods** [comment]
This PR has commits from 5 months ago. Is that correct?

**Comment by ives1227** [comment]
@awoods yes, I did a merge after I created the branch (I had forgotten to do that) and it pulled in a bunch of old stuff.  I would have rebased but I didn't want to break any progress.  This contains everything that it should.

**Review by awoods — CHANGES_REQUESTED** [review]
With the provided `.env`, I am getting "502 Bad Gateway" on the /healthcheck, and 7 tests fail in the container.
```
========================== short test summary info ===========================
FAILED tests/unit/test_batch_builder_assistant.py::test_dvn_run_batch_builder_basic
FAILED tests/unit/test_batch_builder_assistant.py::test_dvn_run_batch_builder_with_overrides
FAILED tests/unit/test_batch_builder_assistant.py::test_dvm_run_batch_builder_basic_doc_only
FAILED tests/unit/test_batch_builder_assistant.py::test_epadd_run_batch_builder_basic
FAILED tests/unit/test_batch_builder_assistant.py::test_epadd_run_batch_builder_with_overrides
FAILED tests/unit/test_batch_builder_assistant.py::test_epadd_with_mods_run_batch_builder_with_overrides
FAILED tests/unit/test_translation_service.py::test_prepare_and_create_mock_lr
================= 7 failed, 30 passed, 16 warnings in 35.77s =================
```

The failed tests appear to be due to a changed method signature:
```
>               raise BatchBuilderException("Failed to create batch, no batch.xml found: " + command)
E               TypeError: TranslationException.__init__() missing 1 required positional argument: 'emailaddress'
```

File: app/__init__.py [hunk 1]
```diff
@@ -60,9 +61,17 @@ def loadreport():
         try:
             load_report_service.handle_load_report(args['filename'], dryrun)
         except LoadReportException as lre:
-            return "Handling of load report failed: {}".format(str(lre)), 400
+            msg = "Handling of load report failed: {}".format(str(lre))
```
[the remark this answers is not in the document]
Comment (ives1227) [reply app/__init__.py:64]: Definitely.  Thanks!

**Comment by ives1227** [comment]
> With the provided `.env`, I am getting "502 Bad Gateway" on the /healthcheck, and 7 tests fail in the container.
> 
> ```
> ========================== short test summary info ===========================
> FAILED tests/unit/test_batch_builder_assistant.py::test_dvn_run_batch_builder_basic
> FAILED tests/unit/test_batch_builder_assistant.py::test_dvn_run_batch_builder_with_overrides
> FAILED tests/unit/test_batch_builder_assistant.py::test_dvm_run_batch_builder_basic_doc_only
> FAILED tests/unit/test_batch_builder_assistant.py::test_epadd_run_batch_builder_basic
> FAILED tests/unit/test_batch_builder_assistant.py::test_epadd_run_batch_builder_with_overrides
> FAILED tests/unit/test_batch_builder_assistant.py::test_epadd_with_mods_run_batch_builder_with_overrides
> FAILED tests/unit/test_translation_service.py::test_prepare_and_create_mock_lr
> ================= 7 failed, 30 passed, 16 warnings in 35.77s =================
> ```
> 
> The failed tests appear to be due to a changed method signature:
> 
> ```
> >               raise BatchBuilderException("Failed to create batch, no batch.xml found: " + command)
> E               TypeError: TranslationException.__init__() missing 1 required positional argument: 'emailaddress'
> ```

@awoods I forgot that you must have BatchBuilder-2.3.2 under the batch_builder_client directory.  You can grab a copy from here: https://drive.google.com/drive/u/2/folders/0Bz4J5tiwltUzYzdTcGl3NC10bFk?resourcekey=0-mnguV0s8lW60H4VaVXoQKg

I've also updated the README to reflect this necessity (sans the drive link)

File: app/load_report_service/load_report.py [hunk 2]
```diff
@@ -3,6 +3,7 @@
 import logging
 from load_report_service.load_report_exception import LoadReportException
 
+logger = logging.getLogger('dts')
```
[the remark this answers is not in the document]
Comment (ives1227) [reply app/load_report_service/load_report.py]: Fixed

**Review by awoods — APPROVED** [review]
All good... healthcheck and pytests pass.

**Pull request merged**